gpiozero/__init__.py

```diff
@@ -70,6 +70,10 @@
     MCP3302,
     MCP3304,
 )
+from .i2c_devices import (
+    I2CDevice,
+    INA219
```

Comment:
Should this mention of INA219 only be in the second commit and not the first one?

Reply:
First of all, thanks for the review @lurch! Sorry I did not see it earler, I somehow missed the notification.
You are right, this was just a sloppy split of the commits. I will fix everything at once in a bit.